assets/DoubleSearchSVG.js

```diff
@@ -0,0 +1,263 @@
+import React from "react";
```

Comment:
What is this file?

Reply:
If I remember correctly, they added an image since the double search bar screen seemed to be a bit too empty. @csbduzi  should be able to confirm.

Reply:
This file contains the svg that is rendered on the DoubleSearch screen :)

Reply:
The image at the bottom?

Reply:
Yes, the svg that was converted into jsx format was held in that file. It renders the image on on the DoubleSearch screen as mentionned by @bkenza

Reply:
yes